en/developer_guide/custom_content_attribute_display.rst

```diff
@@ -0,0 +1,59 @@
+Customize Content Attribute Display
+===================================
+
+Context
+-------
+
+In order to have minimum constraints on contents, Open Orchestra allows you to store any type
+of ``value`` in the ``ContentAttribute``.
+When it comes to display the value, to list the entities for instance, there is no easy way to render
+the value to the backoffice user so we have added a ``stringValue`` to ``ContentAttribute``.
```

Comment:
Saying that stringValue is a class attribute would make the whole doc much clearer.
"we have added" is not ideal, you should say that the class provides a specific attribute to represent the value.